src/inverseRobot.js

```diff
@@ -8,6 +8,21 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inverseArr = {};
+
+  for (const key in robot) {
+    inverseArr[robot[key]] = key;
+  }
+
+  function getObjectLength(obj) {
+    return (Object.keys(obj)).length;
+  }
+
+  if (getObjectLength(robot) !== getObjectLength(inverseArr)) {
+    return null;
+  }
```

Comment:
Do you really need this logic? 
You can just check inside the loop if key is already present in inversedArr as follow:
```suggestion
  if (inversedArr.hasOwnProperty(key)) {
    return null;
  }
```